lib/workers/repository/process/extract-update.ts

```diff
@@ -128,6 +128,10 @@ export async function extract(
         for (const file of files) {
           for (const dep of file.deps) {
             delete dep.updates;
+            if (dep.skipStage && dep.skipStage !== 'extract') {
```

Comment:
so currently it can only be lookup?

Reply:
Or any time package rules apply (was existing)